src/main/java/ch/uzh/ifi/hase/soprafs24/controller/Controller.java

```diff
@@ -57,6 +57,12 @@ public DataDTO getDataByPlayerId(@RequestBody String playerId) {
         return createDataDTO(session, players, tiles);
     }
 
+    @PutMapping("/name")
```

Comment:
Would be nice to have HttpStatus.OK (200) or HttpStatus.NOCONTENT(204) as response.

I dont need it in front-end, so keeping it like this is fine too

Reply:
the default response is HttpStatus.OK

Reply:
Perfect thanks!